source/wp-content/themes/wporg-news-2021/block-template-parts/header-navigation.html

```diff
@@ -0,0 +1,27 @@
+<!-- wp:group {"align":"wide","className":"site-navigation"} -->
```

Comment:
`site-navigation` feels vague enough to be confused with the global header's `site-header__navigation`. should we rename them to something like `global-header__navigation` and `local-header__navigation`?

Reply:
👍🏻 , i'll update the global header in https://github.com/WordPress/wporg-mu-plugins/pull/15